beyond-essence/inventory/toggle-switches-updated.md

```diff
@@ -0,0 +1,127 @@
+* **Allow visitors to make purchases on your website**: 
+When the merchant activates this toggle and saves the changes, a cart will be displayed at the top of the website and each product will have an "Add to cart" button, allowing visitors to order the merchant's products.
+
+💡 _Note: This is a temporary solution. The current implementation needs to be reviewed and updated in the future._
+
+### Enhance cards with extensions
+
+The cockpit is customizable.
+In order to allow merchants to adapt the cockpit to their needs, they can turn features on or off.
+This is possible, for example, on single cards.
+
+**Example: Enable pricing for variations**
+
+For the pricing of variation products, the merchant has two options:
+
+* If the toggle is _disabled_ (default): The same price will be applied to all variations 
+* If the toggle is _enabled_: A table opens on the same card and allows the merchant to set an individual price for each variation.
+
+Both options don't have an impact on the functionality of the other cards, but enhance or reduce the functionality of the Pricing card only.
+
+### Switch between different modes
+
+The third use case for toggle switches in our software is the activation/deactivation of different modes.
+
+**Example: Enable delete mode**
+
+On the Manage variations modal, the merchant can switch between two different modes:
+
+* If the toggle is _disabled_ (default): The modal is in the Edit mode and allows the merchant to make changes to existing variation values.
+* If the toggle is _enabled_: The pen icons on the value bubbles change to x icons allowing the merchant to delete values.
+Also, a confirmation message pops up informing the merchant about the consequences of deleting a variation value.
+
+💡 _Note: This use case is currently not implemented in our software._
+
+## Structure
+
+A toggle switch looks like a classic on/off switch.
+The two different states give clear visual feedback so that the merchant is always aware if the toggle switch is turned on or off.
+
+* A checkmark on a green background, toggled to the right, represents an active feature or mode.
+* A cross on a beige background, toggled to the left, represents a deactivated feature or mode.
+
+On the right side of the toggle switch, a label clearly explains the action that can be taken.
+
+Depending on the complexity of the feature or mode, additional content may be shown below the toggle switch.
+
+{% image_custom image="/assets/img/pages/essence/pricing-for-variations.png" width="100" %}
+
+For further information on the exact visualisation of addtional content, consult the design section.
+
+## Position
+
+Toggle switches may be positioned on a card together with further UI elements that relate to a specific topic.
+
+## Behavior
+
+By selecting either the switch itself or its label, it changes its state:
+
+{% image_custom image="/assets/img/pages/essence/switch-button.gif" width="100" %}
```

Comment:
The gif is showing the old design and needs to be updated. If you toggle the element on the background keeps white:
![image](https://user-images.githubusercontent.com/13624807/98003875-8aeb5580-1def-11eb-9b73-f68ea6801ad6.png)

Reply:
You are right - and it is also outdated, as the one that is shown here is not a toggle anymore. I'll update with another example.